views/npc/Guy view - guest.tw

```diff
@@ -1,4 +1,8 @@
 :: Guy view - guest [sex noredirect girlview] {"position":"5010,145","size":"100,100"}
+<<set $pronounceWhat = setup.pronounceWhat($tmpGirl)>>
```

Comment:
should use underscore instead because, otherwise you're setting global variables.
Underscore variables will work only for that view

<<set _pronounceWhat = setup.pronounceWhat($tmpGirl)>>